modules/quickstart-sap/pages/prepare-clients.adoc

```diff
@@ -0,0 +1,85 @@
+[[quickstart-sap-setup]]
+= Preparing Clients
+
+Your SAP cluster requires several client systems.
+Prepare your clients on physical or virtual hardware, and ensure you have {sles}{nbsp}15 for SAP installation media ready on your {productname} Server.
+You will not be able to create an SAP cluster without the {sles} SAP extension, as it provides tooling specific to SAP.
+
+One of the key features of SAP is high availability of the cluster.
+Every component within an SAP cluster has redundancy and failover protection.
+When you are preparing your clients, ensure you have enough hardware and infrastructure to allow for this.
+For more information about hardware requirements, see https://documentation.suse.com/sles-sap/15-SP1/html/SLES4SAP-quick/cha-s4s-plan.html#sec-s4s-hardware[].
+
+For more information about the clients you need to set up for an SAP cluster, see https://documentation.suse.com/sbp/all[].
+
+
+
+== Register Clients to the {SCC}
+
+Register each client with {scc}, using this command as root, specifying the registration number and the email address associated with it:
+
+----
+SUSEConnect -r <Registration_Number> -e <registered_email_address>
+----
+
+You can get your registration number by logging in to {scc} from your browser.
+
+For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
+
+
+
+== Create the Cluster
+
+Every client system must have all the other client systems listed in their [path]``/etc/hosts`` file.
+Open the [path]``/etc/hosts`` file on each client, and add the hostname for each of the other clients.
+
+
+
+== Create a Shared Storage Device
+
+Each of the clients needs to be able to access a shared disk.
+The shared disk can be physical hardware connected by ethernet, or you can set up a virtual disk and access it with iSCSI.
+
+If you use a virtual disk, consider hosting it on a separate system.
+Do not use a client machine to host the shared storage disk.
+
+
+
+== Download the SAP Installation Software
+
+Download the {sles}{nbsp}15 with SAP installation media from {scc}, and save a copy on each clients.
+Ensure you have saved it in the same file path on each client.
+
+
+
+== Configure Clients to Use Latest ``module.run``
+
+Each client needs to be configured to use the latest version of ``module.run``.
+On the {productname} Server, open the ``/etc/salt/master`` configuration file and add or amend this line:
```

Comment:
If I'm not wrong, this `module.run` configuration must be done in each of the clients. It cannot be done in the `master` machine. Any of the `salt` developers could confirm this?
If this is the case, the file to change would be `/etc/salt/minion`

PD: Now, if I'm not wrong we ship salt 3000, and this version might use already correctly this option